### ISSUE TITLE
python3-mb: ripping without query raises an exception

### ISSUE THREAD

**Issue by pimzand** [jack-cli-cd-ripper/jack#15]
Ripping a CD without metadata available raises an exception

**Comment by madmartin** [comment]
@pimzand Hi,
I cannot confirm this. 
My ~/.jack3rc:
```
# jackrc-version:31
base_dir:/data/jack
encoder:lame
vbr_quality:8.0

```

I just exec "jack" and it rips the CD and it produces output like
```
martin@dj /data/jack $ /home/martin/jack.git.orig/bin/jack 
This is jack 4.0.0 (C)2021 Arne Zellentin, Pim Zandbergen https://github.com/jack-cli-cd-ripper/jack
Options: vbr read-ahead=99 id=sxpC3rz… len=34:47 | press Q to quit
The final status was:
track_01: 6.91x [                              ] [coding @32.3x done, 185kbit
track_02: 6.89x [                              ] [coding @33.0x done, 181kbit
track_03: 7.98x [                              ] [coding @33.3x done, 176kbit
track_04: 8.48x [                              ] [coding @33.4x done, 179kbit
track_05: 8.83x [                              ] [coding @32.1x done, 189kbit
track_06: 9.58x [                              ] [coding @32.4x done, 188kbit
track_07: 10.0x [                              ] [coding @32.3x done, 183kbit
track_08: 9.94x [                              ] [coding @31.7x done, 191kbit
track_09: 9.89x [                              ] [coding @33.3x done, 173kbit
track_10: 10.7x [                              ] [coding @33.1x done, 182kbit
track_11: 11.5x [                              ] [coding @31.7x done, 179kbit
All done. Avg. bitrate: 183kbit
martin@dj /data/jack $ find
.
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/jack.progress
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/jack.toc
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_01.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_02.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_03.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_04.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_05.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_06.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_07.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_08.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_09.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_10.mp3
./jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM-/track_11.mp3
```
No exception.

IMHO works as designed.

**Comment by pimzand** [comment]
It's only occurring when using --server musicbrainz

**Comment by madmartin** [comment]
```
martin@dj /data/jack $ /home/martin/jack.git.orig/bin/jack --server musicbrainz
This is jack 4.0.0 (C)2021 Arne Zellentin, Pim Zandbergen https://github.com/jack-cli-cd-ripper/jack
Options: vbr read-ahead=99 id=sxpC3rz… len=34:47 | press Q to quit
The final status was:
track_01: 6.92x [                              ] [coding @32.3x done, 185kbit
track_02: 6.87x [                              ] [coding @33.0x done, 181kbit
track_03: 8.02x [                              ] [coding @33.4x done, 176kbit
track_04: 8.46x [                              ] [coding @33.5x done, 179kbit
track_05: 8.83x [                              ] [coding @32.2x done, 189kbit
track_06: 9.55x [                              ] [coding @32.1x done, 188kbit
track_07: 9.99x [                              ] [coding @32.2x done, 183kbit
track_08: 9.93x [                              ] [coding @31.9x done, 191kbit
track_09: 9.93x [                              ] [coding @33.2x done, 173kbit
track_10: 10.7x [                              ] [coding @33.2x done, 182kbit
track_11: 11.5x [                              ] [coding @31.7x done, 179kbit
All done. Avg. bitrate: 183kbit
martin@dj /data/jack $ /home/martin/jack.git.orig/bin/jack --server musicbrainz -Q
This is jack 4.0.0 (C)2021 Arne Zellentin, Pim Zandbergen https://github.com/jack-cli-cd-ripper/jack
 *info* matching dir found: /data/jack/jack-sxpC3rzZlv8SjC6yYWVDTFRSyGM- 
 *info* querying... 
Tagging ...........
Done with "Willie Nelson - That's Life".
Year: 2021
Avg. bitrate: 183kbit
martin@dj /data/jack $ 
martin@dj /data/jack $ find
.
./Willie Nelson
./Willie Nelson/That's Life
./Willie Nelson/That's Life/jack.progress
./Willie Nelson/That's Life/jack.toc
./Willie Nelson/That's Life/jack.musicbrainz
./Willie Nelson/That's Life/Willie Nelson - That's Life - 01 - Nice Work If You Can Get It.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 02 - Just In Time.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 03 - A Cottage for Sale.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 04 - I’ve Got You Under My Skin.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 05 - You Make Me Feel So Young.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 06 - I Won’t Dance.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 07 - That’s Life.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 08 - Luck Be a Lady.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 09 - In the Wee Small Hours of the Morning.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 10 - Learnin’ the Blues.mp3
./Willie Nelson/That's Life/Willie Nelson - That's Life - 11 - Lonesome Road.mp3
martin@dj /data/jack $ 
```

Not on my side.
What may be different?

**Comment by pimzand** [comment]
The exception would occur if argument --rename was used, and metadata was not available.

Fixed in commit ee646b61e6b6c727957d4ea879aa98ad50843fa2
